README.md

```diff
@@ -1,13 +1,20 @@
 # Project Happy Thoughts API
 
-Replace this readme with your own information about your project.
+API created to POST and GET thoughts for the Happy Thoughts application - https://myhappy-thoughts.netlify.app/
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+## Feautures
 
-## The problem
+- GET request to get all thoughts
+- POST request to add new thoughts
+- POST request to increase amount of likes
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+## Production Process
```

Comment:
I really like the descriptive process here, makes it easier to understand the code below as well!